runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegistrationView/UsersRegistration.utils.ts

```diff
@@ -0,0 +1,13 @@
+import { UserDto } from 'runbotics-common';
+
+export const getSelectedTenants = (users: UserDto[]) => {
```

Comment:
add return type please

Reply:
If it's an object `{
<userId>: <tenantId>
}`

why is the function name getSelectedTenants?

Reply:
to be consistent with what was defined previously in the UsersRegistartionView. do you have any suggestion to change it?
![image](https://github.com/user-attachments/assets/98658aa2-0e41-4906-971b-b80e51d60eeb)

Reply:
it returns `{ <userId>: <tenantId> }` so I suggest the name - combineUserWithTenantIds or another one that describes what the function do

Also current useState name is a bit confusingf